src/test/java/uk/gov/companieshouse/api/accounts/service/impl/CompanyServiceImplTests.java

```diff
@@ -38,6 +39,9 @@
     @Mock
     private CompanyGet mockCompanyGet;
 
+    @Mock
+    private Transaction mockTransaction;
```

Comment:
This is currently unused, I believe. Can we add unit test coverage for the `isMultipleYearFiler` method please?